README.md

```diff
@@ -51,13 +112,13 @@ docker run --rm --name medusa-proxy -e TORS=3 -e HEADS=2 \
     datawookie/medusa-proxy
 
 # Test SOCKS proxy
-curl --socks5 localhost:5566 http://httpbin.org/ip
+curl --socks5 localhost:1080 http://httpbin.org/ip
 
 # Test HTTP proxy
 curl --proxy localhost:8888 http://httpbin.org/ip
 
 # Run Chromium through the proxy
-chromium --proxy-server="http://localhost:8118" \
+chromium --proxy-server="http://localhost:8888" \
```

Comment:
This change is inconsistent with the ports specified in the `docker run` command.